README.md

```diff
@@ -1,38 +1,46 @@
 # csde-components-boilerplate
 
-Boilerplate project for creating Content Station Digital Editor components.
+Boilerplate project for creating Content Station Digital Editor component sets.
 
-## Setup
+## Creating a component set
 
-Install [NodeJS 8](https://nodejs.org/) or higher.
+This section describes how to set up, develop and package component set out of the boilerplate.
+
+#### *Important: use a release of the boilerplate that is supported by the target Content Station. See the [Releases](https://github.com/WoodWing/csde-components-boilerplate/releases) page for an overview.*
```

Comment:
Would it be possible to change this to a relative link?

Reply:
That's why it took me so long to create the pull request. So far I haven't found a way to do it. Github docs talk about relative paths, but those are within the repo.